Examples/iOSExplorer/iOSExplorer/ODXAppDelegate.m

```diff
@@ -37,9 +37,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // Uncomment the correct call to test with ActiveDirectory or both types of apps.
     [ODClient setMicrosoftAccountAppId:@"0000000048160AF8" scopes:@[@"onedrive.readwrite"] ];
     
-    //[ODClient setActiveDirectoryAppId:<AAD_APPID>
-    //                      redirectURL:<AAD_REDIRECT_URI>
-    //                            flags:@{@"NoThumbnails" : @(YES)}];
+//    [ODClient setActiveDirectoryAppId:<AAD_APPID> /*App ID as registered in your instance of AAD*/
+//                          redirectURL:@"iOSExplorer://com.Microsoft.iOSExplorer" /*URL scheme to relaunch this app, defined in project settings. Must match a Redirect URI registered to this app in AAD.*/
```

Comment:
Is there a default protocol handler registered for the bundle identifier in iOS? I didn't see anywhere in here that your registered a protocol handler.